adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AdalError.cs

```diff
@@ -55,7 +55,7 @@ public static class AdalError
         /// <summary>
         /// Authentication canceled.
         /// </summary>
-        public const string AuthenticationCanceled = "authentication_canceled";
+        public const string AuthenticationCanceled = "user_canceled_authentication";
```

Comment:
Not sure we can change this ne as it's in the public API... I was thinking we needed to add one extra for the new case introduced?